python/ray/scripts/scripts.py

```diff
@@ -443,8 +485,13 @@ def start(node_ip_address, redis_address, address, redis_port, port,
             raise Exception("If --head is not passed in, --redis-max-clients "
                             "must not be provided.")
         if include_webui:
-            raise Exception("If --head is not passed in, the --include-webui "
-                            "flag is not relevant.")
+            raise Exception(
```

Comment:
Try not to raise bare Exceptions, we should probably raise `ValueError` in this case